pkg/operation/care/constraints.go

```diff
@@ -337,6 +337,18 @@ func IsProblematicWebhook(
 	rules []admissionregistrationv1.RuleWithOperations,
 	timeoutSeconds *int32,
 ) bool {
+	// this is a special case, webhook affecting lease in kube-system namespace can block hibernation
+	// even if FailurePolicy is set to `Ignore` and timeoutSeconds is 10 seconds.
+	if timeoutSeconds == nil || (timeoutSeconds != nil && *timeoutSeconds > 3) {
```

Comment:
Can we use constant for `3`, similar to `WebhookMaximumTimeoutSecondsNotProblematic`?